icu4c/source/test/fuzzer/ubidi_fuzzer.cpp

```diff
@@ -0,0 +1,181 @@
+// © 2024 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+// Fuzzer for ICU bidi.
+
+#include <cstring>
+
+#include "fuzzer_utils.h"
+
+#include "unicode/ubidi.h"
+
+#define MAXLEN 200
+
+static void sub2(UBiDi *bidi) {
+  UErrorCode errorCode = U_ZERO_ERROR;
+  int32_t i;
+  int32_t runCount;
+  ubidi_getDirection(bidi);
+  ubidi_getParaLevel(bidi);
+  ubidi_getReorderingMode(bidi);
+  ubidi_getReorderingOptions(bidi);
+  runCount = ubidi_countRuns(bidi, &errorCode);
+  if (U_FAILURE(errorCode)) {
+  } else {
+    for (i = 0; i < runCount; i++) {
+      int32_t start, len;
+      ubidi_getVisualRun(bidi, i, &start, &len);
+    }
+  }
```

Comment:
You're not checking results of anything here.  You're just calling everything to make sure it doesn't crash or corrupt memory?